ext/ForwardDiffGPUArraysCoreExt.jl

```diff
@@ -0,0 +1,62 @@
+module ForwardDiffGPUArraysCoreExt
+
+using GPUArraysCore: AbstractGPUArray
+using ForwardDiff: ForwardDiff, Dual, Partials, npartials, partials
+
+function ForwardDiff.seed!(duals::AbstractGPUArray{Dual{T,V,N}}, x,
+                           seed::Partials{N,V} = zero(Partials{N,V})) where {T,V,N}
+    idxs = collect(ForwardDiff.structural_eachindex(duals, x))
+    duals[idxs] .= Dual{T,V,N}.(x[idxs], Ref(seed))
+    return duals
+end
+
+function ForwardDiff.seed!(duals::AbstractGPUArray{Dual{T,V,N}}, x,
+                           seeds::NTuple{N,Partials{N,V}}) where {T,V,N}
+    idxs = collect(ForwardDiff.structural_eachindex(duals, x))[1:N]
```

Comment:
Maybe just
```suggestion
    idxs = collect(Iterators.take(ForwardDiff.structural_eachindex(duals, x), N))
```
to reduce allocations?

Reply:
Another problem is actually that there could potentially be < N indices.